src/main/java/fi/fmi/avi/model/SpaceWeatherAdvisory/SpaceWeatherAdvisory.java

```diff
@@ -0,0 +1,26 @@
+package fi.fmi.avi.model.SpaceWeatherAdvisory;
+
+import java.util.List;
+import java.util.Optional;
+
+import fi.fmi.avi.model.AviationWeatherMessage;
+
+public interface SpaceWeatherAdvisory extends AviationWeatherMessage {
+    String getIssuingCenterName();
+
+    AdvisoryNumber getAdvisoryNumber();
+
+    Optional<AdvisoryNumber> getReplaceAdvisoryNumber();
+
+    Optional<STATUS> getStatus();
+
+    List<String> getPhenomena();
+
+    List<SpaceWeatherAdvisoryAnalysis> getAnalyses();
+
+    NextAdvisory getNextAdvisory();
+
+    enum STATUS {
+        TEST, EXER
```

Comment:
I would prefer to avoid abbreviations in names, e.g. `EXER` -> `EXERCISE`. It has been the convention throughout this project.